scripts/relabel_nnue_results.py

```
@@ -0,0 +1,135 @@
+            raise ValueError("each FEN rank must contain eight squares")
+    if board.count("K") != 1 or board.count("k") != 1:
+        raise ValueError("FEN must contain exactly one king of each colour")
+    if side not in ("w", "b"):
+        raise ValueError("invalid FEN side to move")
+    if castling != "-" and (any(c not in "KQkq" for c in castling) or len(set(castling)) != len(castling)):
+        raise ValueError("invalid FEN castling rights")
+    if ep != "-" and (len(ep) != 2 or ep[0] not in "abcdefgh" or ep[1] not in "36"):
+        raise ValueError("invalid FEN en-passant square")
+    try:
+        if int(halfmove) < 0 or int(fullmove) < 1:
+            raise ValueError("invalid FEN move clocks")
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)